### PR TITLE
Add gstlal-burst

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for `gstlal-burst`.

cc @myNameIsPatrick as co-maintainer.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by myNameIsPatrick** [comment]
I'm happy to be co-maintainer of this project.

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this is now ready for review, thanks.

File: recipes/gstlal-burst/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    - glib
+    - gobject-introspection >=1.30.0
+    - gsl
+    - gst-plugins-base
+    - gstlal >=1.5.0
+    - gstreamer
+    - lal >=6.19.0
+    - lalburst >=1.5.0
+    - lalinspiral
+    - numpy 1.14
```
Comment (chrisburr) [inline recipes/gstlal-burst/meta.yaml]: Do you need to put a version here? I think the pins are [already 1.14 or higher](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L527-L529):
```suggestion
    - numpy
```

Comment (duncanmmacleod) [reply recipes/gstlal-burst/meta.yaml]: See https://github.com/conda-forge/staged-recipes/pull/10833#discussion_r378228818

**Pull request merged**